### PR TITLE
Update protection-stack-microsoft-defender-for-office365.md

### PR DESCRIPTION
**Pull request by msbemba**
Trusted ARC sealers is an admin-created list of intermediary domains who have implemented ARC sealing. reference https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/use-arc-exceptions-to-mark-trusted-arc-senders?view=o365-worldwide

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8830

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/protection-stack-microsoft-defender-for-office365.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Features in sender intelligence are critical for catching spam, bulk, impersonat
     - **SPF** can reject mails based on DNS TXT records that list IP addresses and servers allowed to send mail on the organization's behalf.
     - **DKIM** provides an encrypted signature that authenticates the sender.
     - **DMARC** lets admins mark SPF and DKIM as required in their domain and enforces alignment between the results of these two technologies.
-    - **ARC** is not customer configured, but builds on DMARC to work with forwarding in mailing lists, while recording an authentication chain.
+    - **ARC**  builds on DMARC to work with forwarding in mailing lists, while recording an authentication chain.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/protection-stack-microsoft-defender-for-office365.md]:
```suggestion
    - **ARC**  builds on DMARC to work with forwarding in mailing lists while recording an authentication chain.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

File: microsoft-365/security/office-365-security/protection-stack-microsoft-defender-for-office365.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Features in sender intelligence are critical for catching spam, bulk, impersonat
     - **SPF** can reject mails based on DNS TXT records that list IP addresses and servers allowed to send mail on the organization's behalf.
     - **DKIM** provides an encrypted signature that authenticates the sender.
     - **DMARC** lets admins mark SPF and DKIM as required in their domain and enforces alignment between the results of these two technologies.
-    - **ARC** is not customer configured, but builds on DMARC to work with forwarding in mailing lists, while recording an authentication chain.
+    - **ARC**  builds on DMARC to work with forwarding in mailing lists while recording an authentication chain.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/protection-stack-microsoft-defender-for-office365.md]:
```suggestion
    - **ARC** builds on DMARC to work with forwarding in mailing lists while recording an authentication chain.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more ,please.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder. thanks

**Pull request merged**